packages/hash/backend-utils/src/logger.ts

```diff
@@ -9,11 +9,26 @@ export type LogLevel = typeof LOG_LEVELS[number];
 
 export type LoggerConfig = {
   mode: "dev" | "prod";
-  level: LogLevel;
+  level?: LogLevel;
   serviceName: string;
   metadata?: Record<string, string>;
 };
 
+const tbdIsLogLevel = (level: string): level is LogLevel =>
+  LOG_LEVELS.includes(level as LogLevel);
+
+const getDefaultLoggerLevel = () => {
+  const envLogLevel = process.env.LOG_LEVEL;
+  return envLogLevel && tbdIsLogLevel(envLogLevel) ? envLogLevel : "info";
+};
+
+const isLogApolloError = ({
+  level,
+  service,
+  message,
+}: winston.Logform.TransformableInfo) =>
+  level === "error" && service === "api" && message === "graphql";
```

Comment:
```suggestion
  level === "error" && service === "api" && message.toLowerCase().includes("graphql");
```
? Could be just my misunderstanding here.

Reply:
Right, I see now. The new approach looks easier to understand, thanks!